src/index.html

```diff
@@ -11,7 +11,108 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form onsubmit="onSubmit()" action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset> <legend>Personal information</legend>
+        <div>
+          <label for="surname">Surname:</label>
+          <input type="surtext" id="surname"
+          name="surname" minlength="2"
+          maxlength="35" autocomplete="off">
+        </div>
+        <div>
+          <label for="name">Name:</label>
+          <input type="text" id="name"
+          name="name" minlength="2"
+          maxlength="35" autocomplete="off">
+        </div>
+        <div>
+          <label for="age">How old are You?</label>
+          <input type="number" id="age"
+          name="age" min="12"
+          max="100" step="5"
+          value="12">
+        </div>
+        <div>
+          <label for="date">Full date of birth:</label>
+          <input type="date" id="date"
+          name="age">
+        </div>
+        <div>
+          <label for="option1">I accept the term of the agreement</label>
+          <input type="checkbox" id="option1"
+        name="option1" value="option1">
+        </div>
+      </fieldset>
+      <fieldset> <legend>Registration</legend>
+        <div>
+          <label for="email">E-mail:</label>
+          <input type="email" id="email"
+          name="email" required
+          placeholder="email@example.com">
+        </div>
+        <div>
+          <label for="password">Password:</label>
+          <input type="password" id="password"
+          name="password">
+        </div>
+      </fieldset>
+
+      <fieldset> <legend>An interesting fact about you!</legend>
+        <div>
+        Do you love cats?
+          <input type="radio" id="yes"
+          name="isHistoryFan" value="yes">
+          <label for="yes">Yes</label>
+          <input type="radio" id="no"
+          name="isHistoryFan" value="no">
+          <label for="no">No</label>
+        </div>
+        <div>
+          <label for="color">What is your favorite color? </label>
+          <input type="color" id="color"
+          name="color">
+        </div>
+        <div>
+          <label for="time">What time do you go to bed?</label>
+          <input type="time" id="time"
+          name="time" step="2">
+        </div>
+        <div>
+          <label for="cars">What are your favorite brand of cars?</label>
+          <select name="cars" id="cars"
+          multiple>
+            <option value="bmw">BMW</option>
+            <option value="saab">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </div>
+        <div>
+          <label for="range">How do you rate our work?</label>
+          <input type="range" id="range"
+          name="range" min="1"
+          max="20" value="1">
+        </div>
+
+      </fieldset>
+      <fieldset> <legend>Additional info:</legend>
```

Comment:
```suggestion
      <fieldset>
        <legend>Additional info:</legend>
```
Один тег - одна строка

Reply:
Дякую, вніс поправки до коду